docs/source/cpp/compute.rst

```diff
@@ -988,44 +988,46 @@ Temporal component extraction
-+--------------------+------------+-------------------+---------------+--------+
-| iso_week           | Unary      | Temporal          | Int64         | \(2)   |
-+--------------------+------------+-------------------+---------------+--------+
-| iso_calendar       | Unary      | Temporal          | Struct        | \(3)   |
-+--------------------+------------+-------------------+---------------+--------+
-| quarter            | Unary      | Temporal          | Int64         |        |
-+--------------------+------------+-------------------+---------------+--------+
-| hour               | Unary      | Temporal          | Int64         |        |
-+--------------------+------------+-------------------+---------------+--------+
-| minute             | Unary      | Temporal          | Int64         |        |
-+--------------------+------------+-------------------+---------------+--------+
-| second             | Unary      | Temporal          | Int64         |        |
-+--------------------+------------+-------------------+---------------+--------+
-| millisecond        | Unary      | Temporal          | Int64         |        |
-+--------------------+------------+-------------------+---------------+--------+
-| microsecond        | Unary      | Temporal          | Int64         |        |
-+--------------------+------------+-------------------+---------------+--------+
-| nanosecond         | Unary      | Temporal          | Int64         |        |
-+--------------------+------------+-------------------+---------------+--------+
-| subsecond          | Unary      | Temporal          | Double        |        |
-+--------------------+------------+-------------------+---------------+--------+
-
-* \(1) Outputs the number of the day of the week. Week begins on Monday and is denoted
-  by 0 and ends on Sunday denoted by 6.
++--------------------+------------+-------------------+---------------+--------+----------------------------+
+| Function name      | Arity      | Input types       | Output type   | Notes  | Options class              |
++====================+============+===================+===============+========+============================+
+| year               | Unary      | Temporal          | Int64         |        |                            |
++--------------------+------------+-------------------+---------------+--------+----------------------------+
+| month              | Unary      | Temporal          | Int64         |        |                            |
++--------------------+------------+-------------------+---------------+--------+----------------------------+
+| day                | Unary      | Temporal          | Int64         |        |                            |
++--------------------+------------+-------------------+---------------+--------+----------------------------+
+| day_of_week        | Unary      | Temporal          | Int64         | \(1)   | :struct:`DayOfWeekOptions` |
```

Comment:
Tiny remark, but can you switch the order of the Notes and Option class columns (so the Notes is the last column). We just had a PR that made this consistent throughout the full document (#10630), so good to keep it consistent now.

Reply:
Indeed. Fixed.